docs/topics/control-flow.md

```diff
@@ -6,20 +6,34 @@ In Kotlin, `if` is an expression: it returns a value.
 Therefore, there is no ternary operator (`condition ? then : else`) because ordinary `if` works fine in this role.
 
 ```kotlin
-var max = a 
-if (a < b) max = b
+fun main() {
+  val a = 2
+  val b = 3
+
+  //sampleStart
+  var max = a
+  if (a < b) max = b
 
-// With else 
-var max: Int
-if (a > b) {
+  // With else
+  if (a > b) {
     max = a
-} else {
+  } else {
     max = b
+  }
+
+  // As expression
+  max = if (a > b) a else b
+
+  // You can also use else if branches in expressions:
```

Comment:
```suggestion
  // You can also use `else if` in expressions:
```

Reply:
I wouldn't call it a branch; it's just an ordinary "if" in the "else" branch.

Reply:
(And I'm sorry, it was my proposal about "else if branches", but after that research (I gave links in Slack), I realized that there are no "else if" branches as in Kotlin, as in Java. Sometimes people call them so but I think we shouldn't to be accurate)